google/cloud/retail/__init__.py

```diff
@@ -59,8 +59,8 @@
 from google.cloud.retail_v2.types.common import FulfillmentInfo
 from google.cloud.retail_v2.types.common import Image
 from google.cloud.retail_v2.types.common import Interval
+from google.cloud.retail_v2.types.common import LocalInventory
 from google.cloud.retail_v2.types.common import PriceInfo
-from google.cloud.retail_v2.types.common import Promotion
```

Comment:
Adding `do not merge` label as this is a breaking change (It would require users to update their code)

Reply:
Thanks Anthonios! I'm aware that this might be a breaking change. But there is currently only one customer using this field and we will communicate with them with this update.
The major reason behind this is that we're planning to update promotion as a new resource in the future so it's better to update it sooner than later. We may bump the major version if that may help mitigate the risks.